sqlite3_func_crypt.go

```diff
@@ -13,24 +13,24 @@ import (
 
 // This file provides several different implementations for the
 // default embedded sqlite_crypt function.
-// This function is uses a caesar-cypher by default
+// This function includes a caesar-cypher by default
 // and is used within the UserAuthentication module to encode
 // the password.
 //
 // The provided functions can be used as an overload to the sqlite_crypt
 // function through the use of the RegisterFunc on the connection.
 //
-// Because the functions can serv a purpose to an end-user
-// without using the UserAuthentication module
-// the functions are default compiled in.
+// Because the functions are useful to an end-user
+// without the UserAuthentication module,
+// the functions are compiled in by default.
 //
 // From SQLITE3 - user-auth.txt
 // The sqlite_user.pw field is encoded by a built-in SQL function
 // "sqlite_crypt(X,Y)".  The two arguments are both BLOBs.  The first argument
 // is the plaintext password supplied to the sqlite3_user_authenticate()
 // interface.  The second argument is the sqlite_user.pw value and is supplied
 // so that the function can extract the "salt" used by the password encoder.
-// The result of sqlite_crypt(X,Y) is another blob which is the value that
+// The result of sqlite_crypt(X,Y) is another blob, which is the value that
```

Comment:
This paragraph is copied directly from here: https://www.sqlite.org/src/doc/trunk/ext/userauth/user-auth.txt

We probably should not edit it.